### ISSUE TITLE
IOError during nbook build, search for html file in docs instead of book

### ISSUE THREAD

**Issue by dlesnoff** [pietroppeter/nimibook#58]
I am trying to build a single-page book. 

## Error output (truncated log)
```raw
nim c -r -d:release nbook.nim build
```
```nim
[...]
[nimib] current directory: /home/kolmogorov/Documents/dev/algoMathBroRemy
[nimib] saving file: book/LogisticSequenceAndChaos.html
entries.nim(14)          check
Error: unhandled exception: Error finding docs/LogisticSequenceAndChaos.html : no such file or directory [IOError]
Error: execution of an external program failed: '/home/kolmogorov/Documents/dev/algoMathBroRemy/nbook build'
```
## My machine
OS: Archlinux

```raw
nim -v
Nim Compiler Version 1.7.3 [Linux: amd64]
Compiled at 2022-10-01
Copyright (c) 2006-2022 by Andreas Rumpf

git hash: 96c5586d030a97d9da2f08b2faacad8d34ebd2e4
active boot switches: -d:release
```
```raw
nimble list -i --ver
nimib  [0.3.1, 0.3.2, #head]
nimibook  [0.2.1, #head]
```
## Book index
```nim
import nimibook

var book = newBookFromToc("Nim for mathematical problems", "book"):
  # section("Dummy", "index"): 
  entry("Logistic Sequences And Chaos", "LogisticSequenceAndChaos.nim")

# access to nimibook cli which will allow to build the book
nimibookCli(book)
```

## Book page ("LogisticSequenceAndChaos.nim")
```nim
import std/[strutils, sequtils, strformat]
import nimib, nimibook
import plotly
import chroma

nbInit(theme = useNimibook)

nbText:"""
# Logistic Sequences and Chaos
"""

nbCode:
  var colors = @[Color(r:0.9, g:0.4, b:0.0, a: 1.0),
                 Color(r:0.9, g:0.4, b:0.2, a: 1.0),
                 Color(r:0.2, g:0.9, b:0.2, a: 1.0),
                 Color(r:0.1, g:0.7, b:0.1, a: 1.0),
                 Color(r:0.0, g:0.5, b:0.1, a: 1.0)]
  var d = Trace[int](mode: PlotMode.LinesMarkers, `type`: PlotType.Scatter)
  var size = @[16.int]
  d.marker =Marker[int](size:size, color: colors)
  d.xs = @[1, 2, 3, 4, 5]
  d.ys = @[1, 2, 1, 9, 5]
  d.text = @["hello", "data-point", "third", "highest", "<b>bold</b>"]

  var layout = Layout(title: "testing", width: 1200, height: 400,
                      xaxis: Axis(title:"my x-axis"),
                      yaxis:Axis(title: "y-axis too"), autosize:false)
  var p = Plot[int](layout:layout, traces: @[d])
  # p.show()

nbSave
```

## Additionnal informations / remarks
This is not very minimal as a MWE (Minimal non Working Example), but I'll reduce the code size when I have more clue why the html has not been generated under the docs folder.

**Comment by HugoGranstrom** [comment]
The fact that it saves the file in the same folder as the sourcefile is interesting? Do you have a `nimib.toml` file in your project? There you set the srcDir (book/ in this case) and homeDir (docs/) if I recall correctly. That should make sure all the files end up at the right place.

**Comment by pietroppeter** [comment]
Did you run the init command? What is in your nimib.toml file (which is created during unit)?

**Comment by dlesnoff** [comment]
I had a toml file before I ran the init command. I cleaned the directory, I have rewritten the nbook.nim file, I have run update many times, and recompiled the nbook.nim file (and went from Nim and nimibook devel to stable for more safety !), and have been able to generate the book.
Btw the docs doesn't say anything about how to render the book ? I launch my browser with the docs/index.html file after ./nbook build.

I think I can close the issue, it will be hard to recreate the issue now. I fixed too much things at the same time to know the exact root cause, sorry.
